process/nettisa.cpp

```diff
@@ -0,0 +1,116 @@
+/**
+ * \file nettisa.cpp
+ * \brief Plugin for creating NetTiSA flow.
+ * \author Josef Koumar koumajos@fit.cvut.cz
+ * \date 2023
+ */
+
+#include <iostream>
+#include <cmath>
+
+#include "nettisa.hpp"
+
+namespace ipxp {
+
+int RecordExtNETTISA::REGISTERED_ID = -1;
+
+__attribute__((constructor)) static void register_this_plugin()
+{
+   static PluginRecord rec = PluginRecord("nettisa", [](){return new NETTISAPlugin();});
+   register_plugin(&rec);
+   RecordExtNETTISA::REGISTERED_ID = register_extension();
+}
+
+
+NETTISAPlugin::NETTISAPlugin()
+{
+}
+
+NETTISAPlugin::~NETTISAPlugin()
+{
+}
+
+void NETTISAPlugin::init(const char *params)
+{
+}
+
+void NETTISAPlugin::close()
+{
+}
+
+ProcessPlugin *NETTISAPlugin::copy()
+{
+   return new NETTISAPlugin(*this);
+}
+
+void NETTISAPlugin::update_record(RecordExtNETTISA *nettisa_data, const Packet &pkt, const Flow &rec)
+{
+   float variation_from_mean = pkt.payload_len_wire - nettisa_data->mean;
+   uint32_t n = rec.dst_packets + rec.src_packets;
+   long diff_time = pkt.ts.tv_usec - nettisa_data->prev_time;
+   nettisa_data->prev_time = pkt.ts.tv_usec;
+   // MEAN
+   nettisa_data->mean += (variation_from_mean) / n;
+   // MIN
+   if(nettisa_data->min > pkt.payload_len_wire)
```

Comment:
```suggestion
   if (nettisa_data->min > pkt.payload_len_wire)
```

Upravit i dale v kodu